packages/tiny_httpd/tiny_httpd.0.7/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+authors: ["Simon Cruanes"]
+maintainer: "simon.cruanes.2007@m4x.org"
+license: "MIT"
+synopsis: "Minimal HTTP server using good old threads"
+build: [
+  ["dune" "build" "@install" "-p" name "-j" jobs]
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+  ["dune" "runtest" "-p" name] {with-test}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name] {with-test & ocaml:version < "4.12"}
```

Reply:
The error is due to a simple missing constraint. I would prefer that over disabling the tests. I'm on it.

Reply:
is there a reason why the test fails on 4.12? I can't think of anything in qcheck_ounit that requires < 4.12?

Reply:
I'm not using stdlib-shims anywhere, I suppose it's a transitive dep that does?

Reply:
Yes, in `ounit2`

Reply:
seems to me that it's either a bug in opam or ounit2's package or travis then. I expect that if my test deps are not compatible with 4.12, then opam should know it's not possible to run tests? 

In other words (sorry, I'm tired): opam should know that the test don't run, imho. I should not have to write constraints that are already expressed in my transitive dependencies.